src/mscorlib/shared/System/StringSpanHelpers.cs

```diff
@@ -125,5 +125,38 @@ public static ReadOnlySpan<char> Remove(this ReadOnlySpan<char> source, int star
             source.Slice(startIndex + count).CopyTo(result.Slice(startIndex));
             return result;
         }
+
+        // Returns the index of the last occurrence of a specified character in the current instance.
+        // The search starts at startIndex and runs backwards to startIndex - count + 1.
+        // The character at position startIndex is included in the search.  startIndex is the larger
+        // index within the string.
+        //
+        public static int LastIndexOf(this ReadOnlySpan<char> source, char value)
```

Comment:
Would these overloads be necessary once we move the MemoryExtensions down (especially the one with startIndex and count)?
https://github.com/dotnet/corefx/blob/master/src/System.Memory/ref/System.Memory.cs#L101-L103

https://github.com/dotnet/corefx/issues/25182

Reply:
That's correct. Many methods in this file are redundant with MemoryExtensions.